src/Components/Subscriptions/index.js

```diff
@@ -1,11 +1,60 @@
+import { useState, useEffect } from 'react';
 import styles from './subscriptions.module.css';
+import Table from './Table';
+import { CreateModal, ErrorModal } from './Modals';
+const Subscriptions = () => {
+  const [subscriptions, setSubscriptions] = useState([]);
+  const [createModal, setCreateModal] = useState(false);
+  const [errorModal, setErrorModal] = useState(false);
+  const [classes, setClasses] = useState([]);
+  const [members, setMembers] = useState([]);
+  // eslint-disable-next-line no-unused-vars
```

Comment:
line 11: We need this line because we're not using the consts we have below: We can take it out, and take it out from the functions we're using it, or use this error in a catch

Reply:
Maybe is easier just to take it out, we're not using it anyway